pkgs/development/compilers/graalvm/default.nix

```diff
@@ -363,12 +376,10 @@ in rec {
       cp -rf ${glibc}/lib/* $out/jre/lib/svm/clibraries/linux-amd64/
       cp ${glibc.static}/lib/* $out/jre/lib/svm/clibraries/linux-amd64/
       cp ${zlib.static}/lib/libz.a $out/jre/lib/svm/clibraries/linux-amd64/libz.a
-      # Organize the out dir
-      mkdir -p $out/share && mv $out/man $out/share
     '';
 
-    dontFixup = true; # do not nuke path of ffmpeg etc
```

Comment:
```dontFixup``` is a rudiment of working with ```oraclejdk8``` binary distribution
fixupPhase does care on man files and also patches ```#!/bin/sh``` shebangs